release-notes/swan-lake-alpha3-release-note.md

```diff
@@ -113,6 +113,19 @@ io:println(nameText); // "DanBrown"
 
 #### Runtime
 
+##### JAVA Runtime API
+
+A new runtime Java API is introduced for creating distinct errors.
+```java
+BError createError(Module module, String errorTypeName, BString message, BError cause, Object details)
+```
+
+With this change, the `createDistinctError` runtime Java API has been deprecated.
```

Comment:
I thought this wasn't just for distinct errors? How about something like
```suggestion
A new runtime Java API is introduced to create errors.
```java
BError createError(Module module, String errorTypeName, BString message, BError cause, Object details)
```

Since distinct errors can be created using this API, the `createDistinctError` API has been deprecated.
```

Reply:
Actually, if you create any errors with createDistinctError, those will not work with hierarchical errors. That's a bug but we can't remove it until usages are removed from stdlibs.

Reply:
How about something like

````suggestion
A new runtime Java API is introduced to create errors.

```java
BError createError(Module module, String errorTypeName, BString message, BError cause, Object details)
```

The `createDistinctError` API has been deprecated and should not be used to create distinct errors. The new `createError` API can be used instead.
````